core/metrics-core-service/src/test/java/org/hawkular/metrics/core/util/GCGraceSecondsManagerITest.java

```diff
@@ -1,145 +0,0 @@
-            return clusterSize;
-        }
-
-        public void setClusterSize(int clusterSize) {
-            this.clusterSize = clusterSize;
-        }
-
-        @Override
-        public void maybeUpdateGCGraceSeconds() {
-            Subscription subscription = null;
-            try {
-                CountDownLatch latch = new CountDownLatch(1);
-                AtomicReference<Throwable> exceptionRef = new AtomicReference<>();
-                subscription = updatesFinished.subscribe(t -> {
-                    exceptionRef.set(t);
-                    latch.countDown();
-                });
-                super.maybeUpdateGCGraceSeconds();
-                latch.await(10, TimeUnit.SECONDS);
-                if (exceptionRef.get() != null) {
-                    fail("Updates failed", exceptionRef.get());
-                }
-            } catch (InterruptedException e) {
-                fail("Timed out waiting for updates to complete", e);
-            } finally {
-                if (subscription != null) {
-                    subscription.unsubscribe();
-                }
-            }
-        }
-    }
-
-    private ConfigurationService configurationService;
-
-    private String keyspace = "gc_grace_test";
-
-    private TestGCGraceSecondsManager manager;
-
-    @BeforeClass
-    public void initClass() {
-        configurationService = new ConfigurationService();
-        configurationService.init(rxSession);
-
-        SchemaService schemaService = new SchemaService();
-        schemaService.run(session, keyspace, Boolean.valueOf(System.getProperty("resetdb", "true")));
-
-        manager = new TestGCGraceSecondsManager(rxSession, keyspace, configurationService);
-        session.getCluster().register(manager);
-    }
-
-    @Test
-    public void singleNodeCluster() throws Exception {
-        manager.maybeUpdateGCGraceSeconds();
-
-        ResultSet resultSet = session.execute(
-                "select table_name, gc_grace_seconds from system_schema.tables where keyspace_name = '" + keyspace +
-                        "'");
-        resultSet.forEach(row -> assertEquals(row.getInt(1), 0, "gc_grace_seconds for " + row.getString(0) +
-                " is wrong"));
-    }
-
-    @Test(dependsOnMethods = "singleNodeCluster")
-    public void multiNodeClusterWithRF2() throws Exception {
-        manager.setClusterSize(3);
-
-        session.execute("alter keyspace " + keyspace + " with replication = {'class': 'SimpleStrategy', " +
-                "'replication_factor': 2}");
-
-        manager.maybeUpdateGCGraceSeconds();
-
-        ResultSet resultSet = session.execute(
-                "select table_name, gc_grace_seconds from system_schema.tables where keyspace_name = '" + keyspace +
-                        "'");
-        resultSet.all().forEach(row -> {
-            if (row.getString(0).equals("data_compressed")) {
-                assertEquals(row.getInt(1), 0, "gc_grace_seconds for " + row.getString(0) + " is wrong");
```

Comment:
Why is this test removed?

Reply:
It was already commented out. And since I knew it had been a source of problems and since I have been having lots of inconsistent test runs on travis, I decided to  altogether remove it. If you fixed it in your other PR, I am perfectly fine with bringing it back.